src/components/shared/Navigation.tsx

```diff
@@ -1,16 +1,251 @@
+                  {isChapter ? "Admin" : npUserName}
+                </Text>
+              </VStack>
+              <MenuButton
+                as={Button}
+                rounded={"full"}
+                variant={"link"}
+                cursor={"pointer"}
+              >
+                <Avatar
+                  size={"sm"}
+                  src={"https://hack4impact.org/brokenLink"}
+                />
+              </MenuButton>
+              <MenuList>
+                <MenuItem>Edit Profile</MenuItem>
+                <MenuItem>Sign Out</MenuItem>
+              </MenuList>
+            </Menu>
+          </Flex>
+        ) : (
+          <Stack
+            flex={{ base: 1, md: 0 }}
+            justify={"flex-end"}
+            direction={"row"}
+            spacing={6}
+          >
+            <Button
+              display={{ base: "none", md: "inline-flex" }}
+              fontSize={"sm"}
+              fontWeight={600}
+              color={"white"}
+              bg={blue}
+              href={"#"}
+              _hover={{
+                bg: "rgba(0, 105, 202, 0.5)",
+              }}
+            >
+              Log In
+            </Button>
+          </Stack>
+        )}
+      </Flex>
+    </Box>
   );
 }
 
 export default Navigation;
+
+const DesktopNav = (props: {
+  isLoggedIn: boolean;
+  isChapter: boolean;
+  isNonprofit: boolean;
+}) => {
+  // const linkColor = useColorModeValue("gray.600", "gray.200");
+  // const linkHoverColor = useColorModeValue("gray.800", "white");
+  // const popoverContentBgColor = useColorModeValue("white", "gray.800");
+
+  //colors
+  const slate = "rgba(101, 119, 136, 1)";
+  const black = "rgba(51, 51, 51, 1)";
+  const blue = "rgba(0, 105, 202, 1)";
+
+  let NAV_ITEMS: Array<NavItem> = [];
+  if (props.isLoggedIn) {
+    if (props.isChapter) {
+      NAV_ITEMS = NAV_ITEMS_ADMIN;
+    } else {
+      NAV_ITEMS = NAV_ITEMS_NON_PROFIT;
+    }
+  }
+  return (
+    <Stack direction={"row"} spacing={6} align="center">
+      {NAV_ITEMS.map((navItem) => (
+        <Box key={navItem.label}>
+          <Popover trigger={"hover"} placement={"bottom-start"}>
+            <PopoverTrigger>
+              <NextLink href={navItem.href ?? "/"}>
+                <Link
+                  p={2}
+                  href={navItem.href ?? "/"}
+                  fontSize={"sm"}
+                  fontWeight={500}
+                  color={slate}
+                  _hover={{
+                    textDecoration: "none",
+                    color: black,
+                  }}
+                  _active={{
+                    fontWeight: 600,
+                    textDecoration: "none",
+                    color: blue,
+                  }}
+                >
+                  {navItem.label}
+                </Link>
+              </NextLink>
+            </PopoverTrigger>
+
+            {/* {navItem.children && ( // not needed rn, maybe later???
```

Comment:
Same as above.  We will not need popovers so you can safely remove the commented code.